react/week4/src/components/UserList.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react";
+import { userListContext } from "../App";
+import EmptyResult from "./EmptyResult";
+
+const UserList = () => {
+  const { user } = useContext(userListContext);
+  return (
+    <ul>
+      {user.total_count === undefined ? (

```

Comment:
This was we cover multiple cases. For example, if our `total_count = 0`, we don't render `EmptyResult` component. With the below solution, we show `EmptyResult` if `total_count` is `0`, `null`, `undefined`, etc.
```suggestion
      {!user.total_count ? (
```